Dockerfile

```diff
@@ -1,4 +1,35 @@
-FROM gcr.io/asylo-framework/asylo:buildenv-v0.5.2
+# Use fixed snapshot of Debian to create a deterministic environment.
+# Snapshot tags can be found at https://hub.docker.com/r/debian/snapshot/tags
+ARG debian_snapshot=buster-20191118
+FROM debian/snapshot:${debian_snapshot}
+
+# Use a fixed version of Bazel.
+ARG bazel_version=1.2.1
```

Comment:
I would keep bazel installation as a separate step after everything else from apt-get, otherwise the entire layer is invalidated when we want to change bazel version.